### PR TITLE
add sem data example

### PR DESCRIPTION
**Pull request by juliuspfadt**

### CODE REVIEW

**Comment by juliuspfadt** [comment]
I didn't mean to update all these other files... can I revert that somehow?

**Comment by shun2wang** [comment]
The most simple way is drop the last commit and re-add the `ADD symptoms` data files.

1. `git reset --hard HEAD~1`
2. add the data files
3. rebase upstreams head: `git rebase upstream/development`
4. froce push to upstream

**Comment by juliuspfadt** [comment]
Thanks, I think I managed somehow :)

**Comment by boutinb** [comment]
Hi @juliuspfadt I have added the json files so that the example will be seen in JASP menu. You still to add a description. There are translations for dutch, german and chinese. I think you can do the dutch and german translations easily. Maybe for the chinese translation, @shun2wang can help you?

File: Resources/Data Sets/index_zh_Hans.json [hunk 1]
```diff
@@ -847,6 +847,13 @@
             "description": "来自 Shrout 和 Fleiss (1979) 关于 ICC 的论文的假设数据集。 该文件演示了经典类内相关系数的使用",
             "kind": "file",
             "associated_datafile": "Interrater Data from Shrout and Fleiss (1979).csv"
```
Comment (shun2wang) [inline Resources/Data Sets/index_zh_Hans.json:849]:
```suggestion
            "associated_datafile": "来自 Shrout 和 Fleiss (1979).csv 评分者数据"
```

**Review by shun2wang — CHANGES_REQUESTED** [review]
~~Ok, I will updated the translate.~~

there is no description should be translated I think? and I woud find a way to translate them with our Weblate hosts later.

**Comment by juliuspfadt** [comment]
I added the descriptions.

**Comment by boutinb** [comment]
> ~Ok, I will updated the translate.~
> 
> there is no description should be translated I think? and I woud find a way to translate them with our Weblate hosts later.

The Data Library description is not yet in WebLate. This should be done before we have Module store. But for the time being, if you can translate the description (Julius just added it) in index_zh_Hans.json, then it will be translated in Chinese.

File: Resources/Data Sets/index_zh_Hans.json [hunk 2]
```diff
@@ -847,6 +847,13 @@
...
+          },
+          {
+            "name": "Add symptoms",
+            "path": "ADD symptoms.jasp",
+            "description": "",
```
Comment (shun2wang) [inline Resources/Data Sets/index_zh_Hans.json]:
```suggestion
            "description": "数据样本由儿童（N=1163）的注意力缺陷障碍 (ADD) 症状进行评分构成，他们按照根据九个5分制李克特量表 (0-4)进行了打分（ Keulers & Hurks, 2021)。有序响应类别 0、1、2、3 、 4 表示特定症状行为的普遍性，范围从 0（从不）到 4（非常频繁）。对于 ADD 量表，总分越高，ADHD 症状越明显。该文件演示了分数相关标准测量误差的使用。",
```

**Review by shun2wang — CHANGES_REQUESTED** [review]

File: Resources/Data Sets/index_zh_Hans.json (hunk 2 above)
Comment (shun2wang) [reply Resources/Data Sets/index_zh_Hans.json]: I just added chinese translations, thanks.

**Comment by shun2wang** [comment]
>The Data Library description is not yet in WebLate. 

Yes I know.I mean I'm looking for a programmatic mechanism from json to po to translate the text, but that has nothing to do with this PR, I just think there might be a way to do it.

**Comment by juliuspfadt** [comment]
I suppose @boutinb could merge it?

**Comment by boutinb** [comment]
> Yes I know.I mean I'm looking for a programmatic mechanism from json to po to translate the text, but that has nothing to do with this PR, I just think there might be a way to do it.

I think we could add the names and the descriptions of the Library files in Description.qml with a new item DataLibrary with actually the same properties as the current index.json file, plus an extra property to tell whether the file should be present also in one or more books:
```
DataLibrary
{
          name: qsTr("Emily Rosa")
          path: "Emily Rosa.jasp"
          description: qsTr("Correct and incorrect decisions from therapeutic touch practitioners in Emily Rosa's famous experiment. \"In the initial trial, the subjects stated the correct location of the investigator's hand in 70 (47%) of 150 tries.\" <br><br> The example JASP file demonstrates the use of a Bayesian binomial test.<br><br> <i>Data set reconstructed from the summary information in Rosa et al. (1998).<\/i>")
          kind: "file",
          associated_datafile: "Emily Rosa.csv"
          books: ["Field"]
}
```

The books should be references to items that could be defined for example in DataLibrary.qml
```
Book
{
    name: "Field"
    label: qsTr("Field - Discovering Statistices"
    description: qsTr("...")
}
``` 

In this way all the string of the Data Library will be automatically in WebLate.

**Comment by shun2wang** [comment]
Yes, good idea, actually the Weblate does [support ](https://docs.weblate.org/en/latest/formats/json.html) `.json` as translation source but I dont know how it works, maybe you can have a look? which may work without refactor I guess.

**Pull request merged**